docker-compose.yml

```diff
@@ -29,6 +29,8 @@ services:
     env_file: ./saleor/common.env
     environment:
       - JAEGER_AGENT_HOST=jaeger
+      - STOREFRONT_URL=http://localhost:3000/
```

Comment:
I'm wondering if this shouldn't be set by the `storefront` service, similarly the dashboard URL by the `dashboard` service. Then these envs would be only set when you actually run the service.

Reply:
Good point.

Reply:
After testing, it can't be done like that.